ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-summary/confirm-page-container-summary.component.js

```diff
@@ -37,9 +50,11 @@ const ConfirmPageContainerSummary = (props) => {
             address={identiconAddress}
           />
         )}
-        <div className="confirm-page-container-summary__title-text">
-          {titleComponent || title}
-        </div>
+        {hideTitle || (
```

Comment:
This is a little different from our convention for conditionally rendering components.

There are a few patterns you can follow:
- `booleanVariable && <Component />`, so that `<Component />` will render if `booleanVariable` is true
- `booleanVariable ? <Component /> : null`, so that `<Component />` will render if `booleanVariable` is true
- `componentVariable1 || somethingElse`, so that a component stored at `componentVariable1` will be rendered if defined or `somethingElse` will be render if `componentVariable1` is not defined

The code here is a little different, where the component only renders if the boolean is false. This will work, but is a new pattern. Sorry that we don't have good documentation on this.

I would recommend the following:

```
{!hideTitle
    ? (<div className="confirm-page-container-summary__title-text">
            {titleComponent || title}
          </div>
        )}
    : null
}

Reply:
@danjm updated